package.json

```diff
@@ -42,7 +42,8 @@
     "@tvkitchen/base-constants": "^1.0.0",
     "@tvkitchen/base-classes": "^1.1.0",
     "@tvkitchen/base-errors": "^1.0.0",
-    "@jest/console": "^25.1.0"
+    "@jest/console": "^25.1.0",
+    "node-fetch": "^2.6.0"
```

Comment:
We've been tending to put new dependencies in their own commit (e.g. "Add node-fetch package") and I think that would be good to continue in this PR.

I appreciate that you explain a bit in this commit as to why we're using node-fetch vs something native in node but a quick extra note would be helpful (like, one sentence; not re-producing an entire "fetch vs http" blog post or whatnot)!

Reply:
I know tweaking history is not a common pattern so here's the steps in case they're handy:

* `git rebase -i`
* mark this commit as "edit"
* create a *NEW* commit (while rebasing) that contains just the package + yarn.lock
* continue the rebase with the rest of the commits

Reply:
Okay.